src/main/java/com/microsoft/sqlserver/jdbc/osgi/Activator.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.jdbc.osgi;
+
+import java.util.Dictionary;
+import java.util.Hashtable;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.jdbc.DataSourceFactory;
+
+import com.microsoft.sqlserver.jdbc.SQLServerDriver;
+
+
+/**
+ * 
+ * publishes the service to the OSGi Framework
```

Comment:
Javadocs: Fix text case and add some content as to what it will be useful for.

Reply:
An activator is the glue between plugin-code and OSGi-Framework